src/propsheet/misc.cpp

```diff
@@ -389,7 +389,7 @@ VOID
 InitializeFonts(
     VOID)
 {
-    EnumerateFonts(EF_DEFFACE);  // Just the Default font
+    (void)EnumerateFonts(EF_DEFFACE);  // Just the Default font

```

Comment:
Uncertain about this.

Reply:
`LOG_IF_FAILED()` would probably be a better approach.